easybuild/easyconfigs/g/goolf/goolf-1.5.16.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "Toolchain"
+
+name = 'goolf'
+version = '1.5.16'
```

Comment:
Strictly speaking, no, it's not necessary.

And since changing the version now would cause problems on your end, let's not go there.